installer/clean

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
What this file is for? It seem to be unrelated to your PR?

Reply:
My bad, I just added all to my commit, I did not edit that file, but I did run the clean.sh file while testing the install.sh file.

Reply:
`git reset --soft head~1` will do the job.